core/src/validator.rs

```diff
@@ -36,17 +43,55 @@ impl From<Error> for ValidationFail {
 /// Validator that verifies the operation is valid.
 ///
 /// Can be upgraded with [`Upgrade`](iroha_data_model::isi::Upgrade) instruction.
-#[derive(DebugCustom, Clone)]
-#[debug(
-    fmt = "Validator {{ loaded_validator: {0:?}, engine: <Engine is truncated> }}",
-    "&self.loaded_validator"
-)]
+#[derive(Debug, Clone, Serialize)]
```

Comment:
That's because of https://github.com/rust-lang/rust/issues/37009. Ideally the only times we need `DebugCustom` are cases where we actually should just `impl core::fmt::Debug`.